Marlin/Configuration.h

```diff
@@ -1498,6 +1498,37 @@
 //
 //#define OLED_PANEL_TINYBOY2
 
+//
+// Makeboard 3D Printer Parts 3D Printer Mini Display 1602 Mini Controller
+// https://www.aliexpress.com/item/Micromake-Makeboard-3D-Printer-Parts-3D-Printer-Mini-Display-1602-Mini-Controller-Compatible-with-Ramps-1/32765887917.html
+//
+// 2 line display.  
+//
+//#define MAKEBOARD_MINI_2_LINE_DISPLAY_1602
+
+//
+// MKS Mini Panel with graphic controller and SD support 
+// http://reprap.org/wiki/MKS_MINI_12864
+//
+//
+//#define MKS_MINI_12864
+
+//
+// Factory display for Creality CR-10 
+// https://www.aliexpress.com/item/Universal-LCD-12864-3D-Printer-Display-Screen-With-Encoder-For-CR-10-CR-7-Model/32833148327.html
+//
+// this is a ramps-compatible version for CR-10 owners who want to replace the Melzi Creality board, but retain the display
```

Comment:
Probably good to include the detail that this is "a RAMPS-compaitible, but with single 10-pin connector instead of 2" so people can more easily identify it (I certainly wasn't aware of that point - great of you to add it here).